workflows/proteomics/clinicalmp/clinicalmp-data-interpretation/WF5_Data_Interpretation_Worklow_tests.yml

```diff
@@ -0,0 +1,67 @@
+- doc: Test outline for WF5_Data_Interpretation_Worklow
+  job:
+    Quantified_Peptides:
+      class: File
+      path: test-data/Quantified_Peptides.tabular
+      filetype: tabular
+    MaxQuant_Protein_Groups:
+      class: File
+      path: test-data/MaxQuant_Protein_Groups.tabular
+      filetype: tabular
+    MaxQuant_Evidence:
+      class: File
+      path: test-data/MaxQuant_Evidence.tabular
+      filetype: tabular
+    Annotation.tabular:
+      class: File
+      path: test-data/Annotation.tabular
+      filetype: tabular
+    Comparison_Matrix.tabular:
+      class: File
+      path: test-data/Comparison_Matrix.tabular
+      filetype: tabular
+  outputs:
+    Human_Proteins:
+      path: test-data/Human_Proteins.tabular
+    Microbial_Proteins:
+      path: test-data/Microbial_Proteins.tabular
+    MICROBIAL_TAXONOMY_TREE:
```

Comment:
Can you remove the files from the test-data folder that are not referenced anymore ?

Reply:
Unnecessary files have been removed from test-data folder